Client/src/Components/InputControls/RadioList.tsx

```diff
@@ -69,6 +70,11 @@ class RadioList extends React.Component<Props> {
                 </HelpIcon>
               }
             </label>
+          {item.body &&
+            <div style={item.value === this.props.value ? {} : {display: "none"}}>
```

Comment:
Can we just use conditional rendering here instead of the `display: none`?

```
{item.body && item.value === this.props.value &&
<div>{item.body}</div>
}
```
or maybe just
```
{item.value === this.props.value && item.body}
```
(assuming undefined/null elements don't render as anything, which I'm pretty sure is the case)

Reply:
I'm not sure what will happen to the item's state! It should disappear, right? If a user selects a dropdown value and clicks away, we want their choice still there, and if I follow this suggestion it might get inserted as a new component.

Reply:
@wbazant is correct that any _component state_ associated with children will be lost. Most likely we will be maintaining that state elsewhere in the application (an ancestor component, redux store, etc), so it might not be applicable. Regardless, there is likely little harm in doing things the current way.